osm2streets/src/lib.rs

```diff
@@ -86,21 +87,36 @@ impl StreetNetwork {
         for i in [id.i1, id.i2] {
             self.intersections.get_mut(&i).unwrap().roads.push(id);
             self.sort_roads(i);
+            // Recalculate movements and complexity.
+            let (complexity, conflict_level, movements) =
+                crate::transform::classify_intersections::guess_complexity(self, &i);
+            let int = self.intersections.get_mut(&i).unwrap();
+            if int.complexity != IntersectionComplexity::MapEdge {
```

Comment:
This feels weird until I read the `guess_complexity` docstring. Feels like it's worth a comment here too (which we can write once if this bit happens in a helper

Reply:
Good call. Maybe `guess_complexity` should be the place that respects `MapEdge`... I'll keep that in mind as we learn more about what `complexity` is actually used for and where movement calculations fit into the overall flow.